### PR TITLE
Added collaboration guidelines

### LINKED ISSUE #10: Add workflow and collaboration instructions
Copied from the slack message:  
1. Fork the repo
2. Clone your fork
3. Add the main repo as a remote (e.g. you can name it upstream)
3. `git checkout -b branch_name` (make sure to use a descriptive name for your branch)
4. Make your changes and commit them under the new branch
5. `git pull --rebase upstream develop`
6. `git push origin branch_name`
7. Open a PR with a nice description.

### PR DESCRIPTION
**Pull request by sadmansk**
Closes #10. @aneedalie Do you mind taking a look?

### CODE REVIEW

**Comment by sadmansk** [comment]
@aneedalie done done. please review

**Comment by aneedalie** [comment]
Looking at it today. Will be done review by tonight.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -336,3 +361,31 @@ The Rule of Zero states that you do not provide any of the functions that the co
...
+
+
+Collaboration Guide
+-------------------
+The basic steps to starting to work on a change and submitting it:
+
+1. Fork the repo
+2. Clone your fork
+3. Add the main repo as a remote (e.g. you can name it upstream)
+3. `git checkout -b branch_name` (make sure to use a descriptive name for your branch)
```
Comment (aneedalie) [inline CONTRIBUTING.md]: `branch_name` to `<branch_name>` to be even more obvious?

Comment (aneedalie) [inline CONTRIBUTING.md]: Also, two `3.`s even though it doesn't show up like that on the actual file.... weird?

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -336,3 +361,31 @@ The Rule of Zero states that you do not provide any of the functions that the co
...
+4. Make your changes and commit them under the new branch
+5. `git pull --rebase upstream develop`
+6. `git push origin branch_name`
```
Comment (aneedalie) [inline CONTRIBUTING.md]: If adding the `<>` in 3 then do it here too.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -336,3 +361,31 @@ The Rule of Zero states that you do not provide any of the functions that the co
...
+7. Open a PR with a nice description.
+8. Make sure the build passes before asking for a review
+
+### Additional Notes:
+Make sure that every single commit messages reflect the purpose of the associated changes. The pull requests should also have relevant names with added description as necessary.
```
Comment (aneedalie) [inline CONTRIBUTING.md]: Grammar: "messages reflect" => "message reflects"

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -336,3 +361,31 @@ The Rule of Zero states that you do not provide any of the functions that the co
...
+
+### Submitting a PR and reviewing:
+The use of proper labels is highly encouraged, as well as using the assignee field
+for tracking task ownership. For example, assign yourself while the issue/PR is
+being worked on, as well as label it with `DO NOT MERGE`. When it's ready for
+review, assign the reviewer, and remove the `DO NOT MERGE` label. After back and
+forths (by changing assignments between the reviewer and reviewee), once the changes
```
Comment (aneedalie) [inline CONTRIBUTING.md]: After ) use "and" instead of ","

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -336,3 +361,31 @@ The Rule of Zero states that you do not provide any of the functions that the co
...
+are approved, the reviewer labels the PR as reviewed and assigns the PR to the
+maintainer so they can merge the PR. After the merge is done, the maintainer will
+assign it back to the author to keep track of ownership in the Kanban board.
```
Comment (aneedalie) [inline CONTRIBUTING.md]: Specify that the Kanban board is the appropriate project in the "Projects" tab?

**Comment by sadmansk** [comment]
@aneedalie finished the changes, can you take a second look? Thanks a lot!

**Comment by sadmansk** [comment]
@aneedalie Thanks a lot! Merging.

**Pull request merged**